piet-cairo/Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["graphics", "2d"]
 categories = ["rendering::graphics-api"]
 
 [dependencies]
-piet = { version = "=0.5.0", path = "../piet" }
+piet = { version = "0.5.1", path = "../piet" }
```

Comment:
you'll notice in all the intra-workspace dependencies we specify them exactly, with `= "=version"`.

...and now I forget why we do this, exactly. It had something to do with the pre-releases for 0.5, where there could be breaking changes between pre-releases and we wanted to make sure anything that specified a pre-release would not be moved from that release by the resolver (since iirc `0.5` is considered compatible with `0.5-pre1` for the purposes of semvar?) 🤔

Reply:
Maybe it's easier for you to just close this PR and do the release, since `cargo ws` automates the whole process.

Reply:
the main blocker for a release is figuring out what to do about #488 😒

Reply:
Aah OK cool. If I get chance I'll have a look at how much work would be required to move to the `windows` crate. It's surely something that will need doing anyway at some point, since it's the one supported by Microsoft.

Reply:
Now that we've decided to ignore the clippy lint, could we do a release now? Worst case scenario, we could yank it, and I don't think that will be necessary.

Reply:
What exactly were you wanting to get in this release? when I diff master against v0.5, I don't see any changes in piet-cairo? (want to make sure I'm not overlooking something)

Reply:
ack I think I made a mistake - it was because I didn't enable the png feature rather than the version. Apologies!